adapter/stackdriver/log/log_test.go

```diff
@@ -195,6 +196,18 @@ func TestHandleLogEntry(t *testing.T) {
 					Payload:   fmt.Sprintf("%d-%s-%v", 1, "foo", now),
 				},
 			}},
+		{"resource",
+			map[string]info{"resource": {tmpl: template.Must(template.New("").Parse("{{.a}}-{{.b}}-{{.c}}")), log: log}},
+			[]*logentry.Instance{{Name: "resource", Variables: map[string]interface{}{"a": 1, "b": "foo", "c": now}, MonitoredResourceType: "mr-type"}},
+			[]logging.Entry{
+				{
+					Timestamp: now,
+					Severity:  logging.Default,
+					Labels:    map[string]string{},
+					Payload:   fmt.Sprintf("%d-%s-%v", 1, "foo", now),
```

Comment:
super-minor-nit: any reason to not just hardcode '1' and `"foo"` in the `fmt` template ?

Reply:
Done